### PR TITLE
adding gir row constraint and removing leftover app attributes;

### PR DESCRIPTION
**Pull request by calellowitz**
This is just some cleanup/followup related to the global impact report.
@dannyroberts

### CODE REVIEW

File: corehq/apps/data_analytics/models.py [hunk 1]
```diff
@@ -77,6 +77,9 @@ class GIRRow(models.Model):
     exp_and_active_ever = models.PositiveIntegerField()
     active_in_span = models.PositiveIntegerField()
 
+    class Meta:
+        unique_together = ('month', 'domain_name')
```
Comment (czue) [inline corehq/apps/data_analytics/models.py]: should there not be a migration for this as well?

Comment (calellowitz) [inline corehq/apps/data_analytics/models.py]: must have neglected to add it to the commit. fixed now

File: corehq/apps/data_analytics/tasks.py [hunk 2]
```diff
@@ -69,10 +69,3 @@ def _last_month_datespan():
         text_content=message
     )
 
-
-@periodic_task(queue='background_queue', run_every=crontab(hour=5, minute=0, day_of_week='*'),
-               ignore_result=True)
-def update_current_GIR():
```
Comment (czue) [inline corehq/apps/data_analytics/tasks.py]: is there some other mechanism that this is getting run from now?

**Comment by czue** [comment]
good to merge on :white_check_mark: from me

**Pull request merged**